### ISSUE TITLE
PHP 5.6.20 and Debian 8

### ISSUE THREAD

**Issue by Florent23** [composer/composer#5425]
Hello,

I try to install composer on my server but i have a multiples errors : 

```
All settings correct for using Composer
PHP Warning:  file_get_contents(): SSL operation failed with code 1. OpenSSL Err                                          or messages:
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify faile                                          d in Command line code(1) : eval()'d code on line 761
PHP Warning:  file_get_contents(): Failed to enable crypto in Command line code(                                          1) : eval()'d code on line 761
PHP Warning:  file_get_contents(https://getcomposer.org/versions): failed to ope                                          n stream: operation failed in Command line code(1) : eval()'d code on line 761
PHP Warning:  Invalid argument supplied for foreach() in Command line code(1) :                                           eval()'d code on line 508
PHP Fatal error:  Uncaught exception 'RuntimeException' with message 'There is n                                          o **version of Composer available for your PHP version (5.6.20-0+deb8u1)' in Comma                                          nd line code(1) : eval()'d code:517**
Stack trace:
#0 Command line code(1) : eval()'d code(55): installComposer(false, false, 'comp                                          oser.phar', false, false, false, 'stable')
#1 Command line code(1) : eval()'d code(13): process(Array)
#2 Command line code(1): eval()
#3 {main}
  thrown in Command line code(1) : eval()'d code on line 517
```

I have try to modify the version of the file, but it doesn't work, anybody have one solution or a version for this combo php & debian ?

**Comment by alcohol** [comment]
> no version of Composer available for your PHP version (5.6.20-0+deb8u1)

Weird. Will have to investigate. In the meanwhile, instead of using the installer, try downloading a Composer binary directly from https://getcomposer.org/download/ (see links at bottom of the page)?

**Comment by Florent23** [comment]
Hello Alcohol,
Thank you for your answer, i have take the last version 1.2-dev and i think it's work :)

> php composer.phar --version
> Composer version 1.2-dev (5a3d60c0cf0302a54b812f26fd23a0db67488e3e) 2016-06-11 14:48:44

I will try that on my project, thank you
